doc/grpc_xds_features.md

```diff
@@ -66,3 +66,4 @@ Support for [xDS v3 APIs](https://www.envoyproxy.io/docs/envoy/latest/api-v3/api
 [Retry](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-retrypolicy):<br>Only the following fields are supported:<ul><li>retry_on for the following conditions: cancelled, deadline-exceeded, internal, resource-exhausted, and unavailable.</li><li>num_retries</li><li>retry_back_off</li></ul> | [A44](https://github.com/grpc/proposal/blob/master/A44-xds-retry.md) | v1.40.0<br>(C++ and Python) | v1.40.1 | 1.41.0 | |
 [Security](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/security):<br>Uses [certificate providers](https://github.com/grpc/proposal/blob/master/A29-xds-tls-security.md#certificate-provider-plugin-framework) instead of SDS | [A29](https://github.com/grpc/proposal/blob/master/A29-xds-tls-security.md) | v1.41.0<br>(C++ and Python) | v1.41.0 | 1.41.0 | |
 [Authorization (RBAC)](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/rbac/v3/rbac.proto):<br><ul><li>`LOG` action has no effect<li>CEL unsupported and rejected</ul> | [A41](https://github.com/grpc/proposal/blob/master/A41-xds-rbac.md) | | v1.42.0 | 1.42.0 | |
+[Outlier Detection](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/outlier):<br>Only the following detection types are supported:<ul><li>Success Rate</li><li>Failure Percentage</li></ul> | [A50](https://github.com/grpc/proposal/blob/master/A50-xds-outlier-detection.md) | v1.49.0 | v1.49.0 | v1.50.0 | v1.7.0
```

Comment:
Unfortunately, I don't think we can quite claim that it was supported in 1.49 of C-core, because it was still env var-protected at that point.  It's still env var-protected in 1.50.

I removed the env var protection in https://github.com/grpc/grpc/pull/31251, but that hasn't made its way into any release yet, so it will be in 1.51.

Reply:
Fixed

Reply:
Saying 1.51 now is a little weird, since that release isn't actually out yet.  But I guess doing it now prevents us from forgetting to do it later.